go.mod

```diff
@@ -14,7 +14,7 @@ require (
 	github.com/alecthomas/units v0.0.0-20190924025748-f65c72e2690d
 	github.com/armon/go-metrics v0.3.3 // indirect
 	github.com/aws/aws-sdk-go v1.35.5
-	github.com/cespare/xxhash v1.1.0
+	github.com/cespare/xxhash/v2 v2.1.2-0.20201007123935-e307e39619f5
```

Comment:
Why pinning a commit vs a release?

Reply:
Just latest (:

Reply:
Is there a good reason not to use 2.1.1? Sticking to a commit means that it will not be updated by `make update-go-dependencies`. Otherwise, can we ask upstream to release 2.1.2?

Reply:
Nah, 2.1.1 is fine. Makes sense - I was always pinning as Go modules were never working correctly (or actually: module authors were never using it properly), so totally did not care about this. Will fix (: